csrc/ir/internal_base_nodes.h

```diff
@@ -458,7 +458,8 @@ class NVF_API TensorDomain : public Val {
       std::vector<IterDomain*> loop_domain,
       std::optional<std::vector<IterDomain*>> alternate_loop_domain,
       std::vector<std::optional<bool>> contiguity = {},
-      std::vector<IterDomain*> additional_ids = {});
+      std::vector<IterDomain*> additional_ids = {},
+      bool skip_checks = false);
```

Comment:
nit: just for consistency since we already have `skip_loop_validation`

```suggestion
      bool skip_validation = false);
```